docs/content/engines/quick_start.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Quick Start"
+weight: 5
+type: docs
+aliases:
+- /engines/quick_start.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+
+# Qucik Start
+This document explains how to use Docker to quickly build a 
+practice environment for Paimon and big data engines (Flink,
+Spark, Hive, etc.).
+
+The following steps have been tested on a Windows 11 laptop 
+with docker desktop installed. 
+
+## Prerequisites
+windows 11
```

Comment:
Why the prerequisites are windows 11? Could not other os system support?

Reply:
I have a windows 11 PC and other systems should be supported. Just follow docker, docker compose

Reply:
In my most recent submission, I added that the answer is yes, but I don't have the right Linux/MacOS computer at hand